### DISCUSSION TITLE
Customize controls?

### DISCUSSION THREAD

**Discussion by nooblyf** [konvajs/konva#1628] [Q&A] [answered]
Is it possible to completely customize controls of Rect, Sprite, Text, etc. I want to make it look something like this

![Screenshot 2023-08-18 at 11 01 59](https://github.com/konvajs/konva/assets/54323745/89ba5c30-2e32-4e10-820c-de50426c3bf6)

**Comment by lavrton** [comment, marked as answer]
This may help: https://konvajs.org/docs/select_and_transform/Transformer_Complex_Styling.html

**Reply by nooblyf** [reply]
Thank you!!
